dbms/src/AggregateFunctions/IAggregateFunction.h

```diff
@@ -119,6 +119,11 @@ class IAggregateFunction
     /// Inserts results into a column.
     virtual void insertResultInto(ConstAggregateDataPtr __restrict place, IColumn & to, Arena * arena) const = 0;
 
+    /// Inserts batch results into a column
+    virtual void insertBatchResultInto(ConstAggregateDataPtr __restrict place, IColumn & to, size_t num, Arena * arena)
```

Comment:
Looks like this method try to insert the same `place` into `to` column for `num` rows.
IMO the method name `insertBatchResultInto` should do things like insert many `places` into a column, just like `IAggregateFunction::addBatch()`.
Maybe you can change to `batchInsertSameResultInto`?

Reply:
Maybe later we will add a new function `batchInsertResultInto()`, which will insert many `places` into a column, this can avoid virtual function call for Aggregation.

Reply:
> batchInsertSameResultInto

done